src/FSharp.Core/list.fsi

```diff
@@ -1478,6 +1657,9 @@ module List =
     [<CompiledName("Max")>]
     val inline max: list:'T list -> 'T when 'T : comparison 
 
+    /// 
+    /// Time complexity: O(n) - where n is the list length
+    /// Space complexity: O(1)
```

Comment:
The added complexity information should be within the `<summary>` tag, right before the closing tag.